static/src/javascripts-legacy/projects/common/modules/commercial/membership-engagement-banner.js

```diff
@@ -34,12 +35,31 @@ define([
                  $,
                  defaults,
                  find,
-                 svgs) {
+                 svgs,
+                 fetch) {
 
         // change messageCode to force redisplay of the message to users who already closed it.
         // messageCode is also consumed by .../test/javascripts/spec/common/commercial/membership-engagement-banner.spec.js
         var messageCode = 'engagement-banner-2017-01-11';
 
+        var SECONDARY_BUTTON_SELECTOR = '.secondary';
+
+        //Remind me form selectors
+        var REMIND_ME_FORM_SELECTOR = '.membership__remind-me-form';
```

Comment:
Why are these variable names capitalised? By convention, some developers capitalise variable names to indicate that the variable is a constant. But some of these variables are re-assigned later on (lines 234-238, line 269)